README.md

```diff
@@ -530,6 +530,7 @@ For those of you who wish for the report to include a custom subset of the avail
     LEAGUE_BAD_BOY_RANKINGS_BOOL=True
     LEAGUE_BEEF_RANKINGS_BOOL=True
     LEAGUE_WEEKLY_TOP_SCORERS_BOOL=True
+    LEAGUE_WEEKLY_LOW_SCORERS_BOOL=True
```

Comment:
Please merge `main`, as this block of text has been deleted from the README so you don't need it anymore.